### PR TITLE
Generating stubs for external libraries

### PR DESCRIPTION
**Pull request by tcarmet**

### CODE REVIEW

File: docs/development/code.md [hunk 1]
```diff
@@ -37,3 +37,32 @@ without having to write boilerplate code.
 processing them in the background with workers.
 
 It will be used to process the jobs that will be created by the runner manager.
+
+## Typing
```
Comment (gaspardmoindrot) [inline docs/development/code.md:41]: Explain what this module is used for.

File: docs/development/code.md [hunk 2]
```diff
@@ -37,3 +37,32 @@ without having to write boilerplate code.
...
+
+`pyright` configuration is located in the
+[`pyrightconfig.json`](../../pyrightconfig.json) file.
+
+It is run automatically when `trunk check` is run.
+
+### Stub files
```
Comment (gaspardmoindrot) [inline docs/development/code.md:48]: Explain what this module is used for.

**Review by gaspardmoindrot — CHANGES_REQUESTED** [review]
Just 2 comments on the documentation. I didn't review the other files since they were all generated by `pyright`, so it wasn't very meaningful.

File: docs/development/code.md (hunk 2 above)
Comment (tcarmet) [reply docs/development/code.md:48]: It's written just below, any particular question?

File: docs/development/code.md (hunk 1 above)
Comment (tcarmet) [reply docs/development/code.md:40]: Already explained, but I added a mini intro here that might make it more clear.

**Review by gaspardmoindrot — APPROVED** [review]

**Pull request merged**